### DISCUSSION TITLE
Voltstro team please help me out here on this issue with communication error

### DISCUSSION THREAD

**Discussion by shubhamnegi1234** [Voltstro-Studios/UnityWebBrowser#165] [Q&A]

**Comment by Voltstro** [comment]
Hi. Please don't open multiple discussions for the same thing you reported yesterday.

See #153. [Someone there deleted their `Library` folder](https://github.com/Voltstro-Studios/UnityWebBrowser/issues/153#issuecomment-1475416388), and it seems to fix it for them.

**Reply by shubhamnegi1234** [reply]
i am sorry for this i am ne on this , about above solution it worked for me as well but i have to repeat the solution again and again can you provide some permanent solution ?
please ?
my job depends upon this problem

**Reply by Voltstro** [reply]
I have some other issues/general cleanup that I was going to do this weekend, so I can do it then, but no grantees I can fix this particular issue, this issue has been reported over and over again (#158 #155 #161), and I my-self have never been able to replicate it.

One of the issues that I want to fix is with exceptions not properly logging, so hopefully that will help to find out why the engine process keeps crashing and fix this issue.

**Reply by Voltstro** [reply]
I am consolidating everything related to this issue into #166. Please see that issue report.
